model/menuStructure/Action.php

```diff
@@ -103,4 +109,9 @@ public function isMultiple();
      * @return array
      */
     public function getRequiredRights();
+
+    /**
+     * @return int Weight of the action
+     */
+    public function getWeight();
```

Comment:
This one should be committed with `feat!: your commit text` as you are adding breaking changes  https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/872775905/Conventional+Commits